example/package.json

```diff
@@ -5,16 +5,18 @@
   "scripts": {
     "start": "node ../node_modules/react-native/local-cli/cli.js start",
     "run:android": "react-native run-android",
+    "run:windows": "react-native start --use-react-native-windows --projectRoot ../",
```

Comment:
why not react-native run-windows?

Reply:
copied from WebView, which we link to in our docs (https://microsoft.github.io/react-native-windows/docs/native-modules-setup#option-2-adding-windows-support-to-existing-sample-app)

Reply:
changed.